routes/impersonation.ts

```
@@ -0,0 +1,9 @@
+import express from "express";
+const router = express.Router();
+import authenticate from "../middlewares/authenticate";
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)